crates/autopilot/src/arguments.rs

```diff
@@ -56,6 +56,10 @@ pub struct Arguments {
     #[clap(long, env, default_value = "500")]
     pub order_events_insert_batch_size: NonZeroUsize,
 
+    /// The number of fee policies to insert in a single batch.
+    #[clap(long, env, default_value = "500")]
+    pub fee_policies_insert_batch_size: NonZeroUsize,
```

Comment:
Can we move this down to be part of the `FeePolicy` group of parameters.

`pub fee_policy: FeePolicy,`

Reply:
Done